src/Bicep.RegistryModuleTool/JsonSchemas/schema.metadata.json

```diff
@@ -3,7 +3,7 @@
     "$schema": {
       "type": "string",
       "enum": [
-        "https://aka.ms/bicep-registry-module-metadata-file-schema#"
+        "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#"
```

Comment:
The old aka link was updated to reference a specific commit on the main branch, so the change won't affect old versions.